docs/run-integration-tests.md

```diff
@@ -0,0 +1,34 @@
+# Run integration tests
+
+## Writing tests and running them locally
+
+1. Create `.env` file at the root of the repository with the following variables:
+
+```bash
+CONTENTFUL_CMA_TOKEN=<personal-cma-token>
```

Comment:
Link to https://www.contentful.com/r/knowledgebase/personal-access-tokens/ in a comment